#
# Camera output
#

# Header
Header header

# Coordinates of the data points (in camera frame)
float32[] x
float32[] y
float32[] z

# Distance between the data points and the origin of the viewpoint
float32[] distance

# Value of the data points
float32[] value
